src/Test/HttpClient.php

```diff
@@ -113,7 +113,6 @@ private function createRequest($method, $uri, $headers)
         return MessageFactoryDiscovery::find()->createRequest(
             $method,
             $uri,
-            '1.1',
```

Comment:
this was reordered in the factory to avoid having to specify it when setting headers.